app/src/pages/OnDeviceDisplay/ProtocolDetails/index.tsx

```diff
@@ -48,26 +50,36 @@ import { Hardware } from './Hardware'
 import { Labware } from './Labware'
 import { Liquids } from './Liquids'
 
+import type { Protocol } from '@opentrons/api-client'
 import type { ModalHeaderBaseProps } from '../../../molecules/Modal/types'
 import type { Dispatch } from '../../../redux/types'
 import type { OnDeviceRouteParams } from '../../../App/types'
 import { useOffsetCandidatesForAnalysis } from '../../../organisms/ApplyHistoricOffsets/hooks/useOffsetCandidatesForAnalysis'
 
-const ProtocolHeader = (props: {
-  title: string
+interface ProtocolHeaderProps {
+  title: string | null | undefined
```

Comment:
I recommend adding the `?` instead of writing `undefined` (that is typically what we do now a days). Also, does `title` need to be both `null` AND `undefined` maybe we can remove the `undefined`? 

```suggestion
  title?: string | null
```